models/Product.js

```diff
@@ -10,13 +10,45 @@ class Product extends Model {}
 Product.init(
   {
     // define columns
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      allowNull: false,
+      autoIncrement: true,
+    },
+    product_name: {
+      type: DataTypes.STRING(120),
+      allowNull: false,
+    },
+    price: {
+      type: DataTypes.DECIMAL,
+      allowNull: false,
+      validate: {
+        isDecimal: true,
+      },
+    },
+    stock: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      defaultValue: 10,
+      validate: {
+        isNumeric: true,
+      },
+    },
+    category_id: {
+      type: DataTypes.INTEGER,
+      references: {
+        model: "category",
+        key: "id",
+      },
+    },
```

Comment:
Just as an FYI, Sequelize also offers onDelete and onUpdate options so that if you ever wanted a product to be deleted/set to null/not changed if its category is deleted you could specify that here.

You can find info on the different options here: https://sequelize.org/v5/class/lib/model.js~Model.html

Reply:
Just seen you actually specified these in your model associations, nice job! You can do this here also, there's no right or wrong way though :D